src/components/Page/PageIcon.module.css

```diff
@@ -0,0 +1,24 @@
+.page-icon {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  margin-left: 1.8rem;
+}
+
+@media only screen and (min-width: 0) {
+  .page-icon__element {
+    width: 28px;
+    height: 28px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .page-icon__element {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.page-icon__element--primary {
+  fill: white;
```

Comment:
This can be avoided entirely, by setting `fill: currentColor` on the `.page-icon` class itself. It will then follow whatever color is set on the text, and from what I've seen in the mockups this is the behavior we want (icon color and text color is the same). By doing this, we wont need to add modifiers for each variant.

Reply:
Like this? I didn't get it to work
image.png

Reply:
Like this? I didn't get it to work
<img width="1688" alt="image" src="https://user-images.githubusercontent.com/31471142/190363599-4a4b5896-dcdc-450f-84c2-611f780824a1.png">

Reply:
That is because the `color: white` property is set on `.page-header__title` and not on `.page-header` which I assumed it was. Is there any reason why we cannot move this definition? Any text in the header should probably inherit the variant color?

Reply:
Nope, I'll move it.